zcash_proofs/src/lib.rs

```diff
@@ -74,15 +76,15 @@ pub fn load_parameters(
             .expect("couldn't finish reading Sprout groth16 parameter file");
     }
 
-    if spend_fs.into_hash() != spend_hash {
+    if spend_fs.into_hash() != SAPLING_SPEND_HASH {
         panic!("Sapling spend parameter file is not correct, please clean your `~/.zcash-params/` and re-run `fetch-params`.");
     }
 
-    if output_fs.into_hash() != output_hash {
+    if output_fs.into_hash() != SAPLING_OUTPUT_HASH {
         panic!("Sapling output parameter file is not correct, please clean your `~/.zcash-params/` and re-run `fetch-params`.");
     }
 
-    if sprout_fs.map(|fs| fs.into_hash()) != sprout_hash.map(|h| h.to_owned()) {
+    if sprout_fs.map(|fs| fs.into_hash()) != Some(SPROUT_HASH.to_owned()) {
```

Comment:
I believe this now requires Sprout parameters to be provided, as this will evaluate to `true` if `sprout_fs` is `None`. I'm surprised that there aren't any failing tests... In any case, this should be:
```suggestion
    if sprout_fs.map(|fs| fs.into_hash() != SPROUT_HASH).unwrap_or(false) {
```